### ISSUE TITLE
Tab order focus should be reordered

### ISSUE THREAD

**Issue by marisademeglio** [daisy/pipeline-ui#8]
"Open Pipeline, select any script, Press Tab button focus goes to Run button, Press the button, textbox throws error 'Please fill out this field', ideally the process should be focus goes to Browse button with textbox then RUN button."

cc @rachanasinghg

**Comment by marisademeglio** [comment]
Ok. I can see how it's causing an extra tab press to get to the form fields.

Just thinking aloud here - I like having the Run button at the top like that, so it's visually prominent (easy to see what you're supposed to do).

We could force the tab order to go through all the form fields first before reaching 'run' but it would require giving the other items a tabindex=1 value, which is not recommended.

**Comment by avneeshsingh** [comment]
The current order of tabs do not look intuetive to screen reader users. Screen readers users can realize the control which has the keyboard focus. So, it looks strange when I hit run button as first item on the page. A visual user can make out at first glance that there are more things to do before pressing run button. But it is not so straight forward for screen reader user to realize. So, fixing this is high priority.
